src/gcp_scanner/arguments.py

```diff
@@ -127,9 +127,12 @@ def arg_parser():
 
   args: argparse.Namespace = parser.parse_args()
 
-  if not args.key_path and not args.gcloud_profile_path \
-    and not args.use_metadata and not args.access_token_files\
-    and not args.refresh_token_files:
+  if not list(filter(lambda arg: bool(arg) is True,
```

Comment:
Please use a list comprehension instead of filter+lambda

Reply:
good point